var/spack/repos/builtin/packages/py-proj/package.py

```diff
@@ -0,0 +1,28 @@
+from spack import *
+
+class PyProj(Package):
+    """Python interface to the PROJ.4 Library."""
+    homepage = "http://jswhit.github.io/pyproj/"
+    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
+
+
+    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
+
+    # We need the benefits of this PR
+    # https://github.com/jswhit/pyproj/pull/54
+    version('citibeth-latlong2',
+        git='https://github.com/citibeth/pyproj.git',
+        branch='latlong2')
+
+    extends('python')
+
+    depends_on('py-cython')
+    depends_on('py-setuptools')
```

Comment:
GitHub formatted the PR in a funny way.  But at the end of the file in the
PR (not quoted above), there's a note explaining this:

```
+    # The py-proj git repo actually includes the correct version of PROJ.4,
+    # which is built internally as part of the py-proj build.
+    # Adding depends_on('proj') will cause mysterious build errors.
+
+    def install(self, spec, prefix):
+        python('setup.py', 'install', '--prefix=%s' % prefix)
```

-- Elizabeth

On Tue, Apr 5, 2016 at 11:17 AM, Ben Boeckel <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/py-proj/package.py:
>
> > +    homepage = "http://jswhit.github.io/pyproj/"
> > +    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
> > +
> > +
> > +    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
> > +
> > +    # We need the benefits of this PR
> > +    # https://github.com/jswhit/pyproj/pull/54
> > +    version('citibeth-latlong2',
> > +        git='https://github.com/citibeth/pyproj.git',
> > +        branch='latlong2')
> > +
> > +    extends('python')
> > +
> > +    depends_on('py-cython')
> > +    depends_on('py-setuptools')
>
> This doesn't depends_on the proj library?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LLNL/spack/pull/717/files/552a2815cd82b04c74eac44352f5e812ea0fa7f6#r58556930>
>